examples/with_flags.rs

```diff
@@ -0,0 +1,20 @@
+//! This example shows how to use the `group_spawn` method
```

Comment:
This doesn't use group_spawn, either say "the group() builder" or "CommandGroup", but it shouldn't mention the wrong method

Reply:
Oops, some copy-pasting for the examples...